### ISSUE TITLE
[2.8.26] Service Interleave Factor is reset to 2 by Centreon Web

### ISSUE THREAD

**Issue by kurdbahr** [centreon/centreon#7155]
The Centreon Web UI resets the Service Interleave Factor for Centreon Engine automagically back to '2' whenever I try to change it.

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**: 2.8.26
**Centreon Engine version**: 1.8.1
**Centreon Broker version**: 3.0.16
**OS**: CentOS Linux release 7.6.1810 (Core)

**Steps to reproduce the issue:**
1. Go to Configuration / Pollers / Engine Configuration / Tuning
2. Change Service Interleave Factor to something else than '2', for example 's'
3. Save.
4. Open same configuration tab again.

**Describe the results you received:**
Service Interleave Factor is set back to '2' again.

**Describe the results you expected:**
The value entered above ('s').

**Additional information you think important (e.g. issue happens only occasionally):**
Other changes, e.g. Maximum Number of Service Checks, can be configured as expected.

**Comment by mavynet** [comment]
The problem seems to be in  /usr/share/centreon/www/include/configuration/configNagios/DB-Func.php

$ret["service_interleave_factor"]["service_interleave_factor"] should be just **$ret["service_interleave_factor"]**

This fixed it for me.

**Comment by kurdbahr** [comment]
Thanks for the hint @mavynet. This also fixed it for me.

There are many other places in DB-Func.php where the same kind of "double-indexed" array is used, where it probably should be just a simple ```$ret["something"]```.

Maybe someone (with more PHP experience than me) can check this:

```
# fgrep '"]["' DB-Func.php
    isset($ret["enable_notifications"]["enable_notifications"])
        && $ret["enable_notifications"]["enable_notifications"] != 2 ?
        $rq .= "'".$ret["enable_notifications"]["enable_notifications"]."',  " : $rq .= "'2', ";
    isset($ret["execute_service_checks"]["execute_service_checks"])
        && $ret["execute_service_checks"]["execute_service_checks"] != 2 ?
        $rq .= "'".$ret["execute_service_checks"]["execute_service_checks"]."',  " : $rq .= "'2', ";
    isset($ret["accept_passive_service_checks"]["accept_passive_service_checks"])
        && $ret["accept_passive_service_checks"]["accept_passive_service_checks"] != 2 ?
        $rq .= "'".$ret["accept_passive_service_checks"]["accept_passive_service_checks"]."',  " : $rq .= "'2', ";
    isset($ret["execute_host_checks"]["execute_host_checks"])
        && $ret["execute_host_checks"]["execute_host_checks"] != 2 ?
        $rq .= "'".$ret["execute_host_checks"]["execute_host_checks"]."',  " : $rq .= "'2', ";
    isset($ret["accept_passive_host_checks"]["accept_passive_host_checks"])
        && $ret["accept_passive_host_checks"]["accept_passive_host_checks"] != 2 ?
        $rq .= "'".$ret["accept_passive_host_checks"]["accept_passive_host_checks"]."',  " : $rq .= "'2', ";
    isset($ret["enable_event_handlers"]["enable_event_handlers"])
        && $ret["enable_event_handlers"]["enable_event_handlers"] != 2 ?
        $rq .= "'".$ret["enable_event_handlers"]["enable_event_handlers"]."',  " : $rq .= "'2', ";
    isset($ret["log_rotation_method"]["log_rotation_method"])
        && $ret["log_rotation_method"]["log_rotation_method"] != 2 ?
        $rq .= "'".$ret["log_rotation_method"]["log_rotation_method"]."',  " : $rq .= "'2', ";
    isset($ret["check_external_commands"]["check_external_commands"])
        && $ret["check_external_commands"]["check_external_commands"] != 2 ?
        $rq .= "'".$ret["check_external_commands"]["check_external_commands"]."',  " : $rq .= "'2', ";
    isset($ret["retain_state_information"]["retain_state_information"])
        && $ret["retain_state_information"]["retain_state_information"] != 2 ?
        $rq .= "'".$ret["retain_state_information"]["retain_state_information"]."',  " : $rq .= "'2', ";
    isset($ret["use_retained_program_state"]["use_retained_program_state"])
        && $ret["use_retained_program_state"]["use_retained_program_state"] != 2 ?
        $rq .= "'".$ret["use_retained_program_state"]["use_retained_program_state"]."',  " : $rq .= "'2', ";
    isset($ret["use_retained_scheduling_info"]["use_retained_scheduling_info"])
        && $ret["use_retained_scheduling_info"]["use_retained_scheduling_info"] != 2 ?
        $rq .= "'".$ret["use_retained_scheduling_info"]["use_retained_scheduling_info"]."',  " : $rq .= "'2', ";
    isset($ret["use_syslog"]["use_syslog"]) && $ret["use_syslog"]["use_syslog"] != 2 ?
        $rq .= "'".$ret["use_syslog"]["use_syslog"]."',  " : $rq .= "'2', ";
    isset($ret["log_notifications"]["log_notifications"]) && $ret["log_notifications"]["log_notifications"] != 2 ?
        $rq .= "'".$ret["log_notifications"]["log_notifications"]."',  " : $rq .= "'2', ";
    isset($ret["log_service_retries"]["log_service_retries"])
        && $ret["log_service_retries"]["log_service_retries"] != 2 ?
        $rq .= "'".$ret["log_service_retries"]["log_service_retries"]."',  " : $rq .= "'2', ";
    isset($ret["log_host_retries"]["log_host_retries"])
        && $ret["log_host_retries"]["log_host_retries"] != 2 ?
        $rq .= "'".$ret["log_host_retries"]["log_host_retries"]."',  " : $rq .= "'2', ";
    isset($ret["log_event_handlers"]["log_event_handlers"])
        && $ret["log_event_handlers"]["log_event_handlers"] != 2 ?
        $rq .= "'".$ret["log_event_handlers"]["log_event_handlers"]."',  " : $rq .= "'2', ";
    isset($ret["log_external_commands"]["log_external_commands"])
        && $ret["log_external_commands"]["log_external_commands"] != 2 ?
        $rq .= "'".$ret["log_external_commands"]["log_external_commands"]."',  " : $rq .= "'2', ";
    isset($ret["log_passive_checks"]["log_passive_checks"])
        && $ret["log_passive_checks"]["log_passive_checks"] != 2 ?
        $rq .= "'".$ret["log_passive_checks"]["log_passive_checks"]."',  " : $rq .= "'2', ";
    isset($ret["auto_reschedule_checks"]["auto_reschedule_checks"])
        && $ret["auto_reschedule_checks"]["auto_reschedule_checks"] != 2 ?
        $rq .= "'".$ret["auto_reschedule_checks"]["auto_reschedule_checks"]."', " : $rq .= "'2', ";
    isset($ret["use_aggressive_host_checking"]["use_aggressive_host_checking"])
        && $ret["use_aggressive_host_checking"]["use_aggressive_host_checking"] != 0 ?
        $rq .= "'".$ret["use_aggressive_host_checking"]["use_aggressive_host_checking"]."',  " : $rq .= "'0', ";
    isset($ret["enable_predictive_host_dependency_checks"]["enable_predictive_host_dependency_checks"])
        && $ret["enable_predictive_host_dependency_checks"]["enable_predictive_host_dependency_checks"] != 2 ?
        $rq .= "'".$ret["enable_predictive_host_dependency_checks"]["enable_predictive_host_dependency_checks"]."',  "
    isset($ret["enable_flap_detection"]["enable_flap_detection"])
        && $ret["enable_flap_detection"]["enable_flap_detection"] != 2 ?
        $rq .= "'".$ret["enable_flap_detection"]["enable_flap_detection"]."',  " : $rq .= "'2', ";
    isset($ret["soft_state_dependencies"]["soft_state_dependencies"])
        && $ret["soft_state_dependencies"]["soft_state_dependencies"] != 2 ?
        $rq .= "'".$ret["soft_state_dependencies"]["soft_state_dependencies"]."',  " : $rq .= "'2', ";
    isset($ret["enable_predictive_service_dependency_checks"]["enable_predictive_service_dependency_checks"])
        && $ret["enable_predictive_service_dependency_checks"]["enable_predictive_service_dependency_checks"] != 2 ?
        $rq .= "'".$ret["enable_predictive_service_dependency_checks"]["enable_predictive_service_dependency_checks"]
    isset($ret["obsess_over_services"]["obsess_over_services"])
        && $ret["obsess_over_services"]["obsess_over_services"] != 2 ?
        $rq .= "'".$ret["obsess_over_services"]["obsess_over_services"]."',  " : $rq .= "'2', ";
    isset($ret["obsess_over_hosts"]["obsess_over_hosts"])
        && $ret["obsess_over_hosts"]["obsess_over_hosts"] != 2 ?
        $rq .= "'".$ret["obsess_over_hosts"]["obsess_over_hosts"]."',  " : $rq .= "'2', ";
    isset($ret["process_performance_data"]["process_performance_data"])
        && $ret["process_performance_data"]["process_performance_data"] != 2 ?
        $rq .= "'".$ret["process_performance_data"]["process_performance_data"]."',  " : $rq .= "'2', ";
    isset($ret["host_perfdata_file_mode"]["host_perfdata_file_mode"])
        && $ret["host_perfdata_file_mode"]["host_perfdata_file_mode"] != null ?
        $rq .= "'".$ret["host_perfdata_file_mode"]["host_perfdata_file_mode"]."',  " : $rq .= "NULL, ";
    isset($ret["service_perfdata_file_mode"]["service_perfdata_file_mode"])
        && $ret["service_perfdata_file_mode"]["service_perfdata_file_mode"] != null ?
        $rq .= "'".$ret["service_perfdata_file_mode"]["service_perfdata_file_mode"]."',  " : $rq .= "NULL, ";
    isset($ret["check_for_orphaned_services"]["check_for_orphaned_services"])
        && $ret["check_for_orphaned_services"]["check_for_orphaned_services"] != 2 ?
        $rq .= "'".$ret["check_for_orphaned_services"]["check_for_orphaned_services"]."',  " : $rq .= "'2', ";
    isset($ret["check_service_freshness"]["check_service_freshness"])
        && $ret["check_service_freshness"]["check_service_freshness"] != 2 ?
        $rq .= "'".$ret["check_service_freshness"]["check_service_freshness"]."',  " : $rq .= "'2', ";
    isset($ret["check_host_freshness"]["check_host_freshness"])
        && $ret["check_host_freshness"]["check_host_freshness"] != 2 ?
        $rq .= "'".$ret["check_host_freshness"]["check_host_freshness"]."',  " : $rq .= "'2', ";
    isset($ret["use_large_installation_tweaks"]["use_large_installation_tweaks"])
        && $ret["use_large_installation_tweaks"]["use_large_installation_tweaks"] != 2 ?
        $rq .= "'".$ret["use_large_installation_tweaks"]["use_large_installation_tweaks"]."',  " : $rq .= "'2', ";
    isset($ret["debug_verbosity"]["debug_verbosity"])
        && $ret["debug_verbosity"]["debug_verbosity"] != 2 ?
        $rq .= "'".$ret["debug_verbosity"]["debug_verbosity"]."',  " : $rq .= "'2', ";
    isset($ret["daemon_dumps_core"]["daemon_dumps_core"])
        && $ret["daemon_dumps_core"]["daemon_dumps_core"] ?
    isset($ret["enable_environment_macros"]["enable_environment_macros"])
        && $ret["enable_environment_macros"]["enable_environment_macros"] != 2 ?
        $rq .= "'".$ret["enable_environment_macros"]["enable_environment_macros"]."',  " : $rq .= "'2', ";
    isset($ret["use_setpgid"]["use_setpgid"]) && $ret["use_setpgid"]["use_setpgid"] != 2 ?
        $rq .= "'".$ret["use_setpgid"]["use_setpgid"]."',  " : $rq .= "'2', ";
    isset($ret["use_regexp_matching"]["use_regexp_matching"])
        && $ret["use_regexp_matching"]["use_regexp_matching"] != 2 ?
        $rq .= "'".$ret["use_regexp_matching"]["use_regexp_matching"]."',  " : $rq .= "'2', ";
    isset($ret["use_true_regexp_matching"]["use_true_regexp_matching"])
        && $ret["use_true_regexp_matching"]["use_true_regexp_matching"] != 2 ?
        $rq .= "'".$ret["use_true_regexp_matching"]["use_true_regexp_matching"]."',  " : $rq .= "'2', ";
    isset($ret["nagios_activate"]["nagios_activate"])
        && $ret["nagios_activate"]["nagios_activate"] != null ?
        $rq .= "'".$ret["nagios_activate"]["nagios_activate"]."'," : $rq .= "'0',";
    isset($ret["translate_passive_host_checks"]["translate_passive_host_checks"])
        && $ret["translate_passive_host_checks"]["translate_passive_host_checks"] != 2 ?
        $rq .= "'".$ret["translate_passive_host_checks"]["translate_passive_host_checks"]."', " : $rq .= "'2', ";
    isset($ret["passive_host_checks_are_soft"]["passive_host_checks_are_soft"])
        && $ret["passive_host_checks_are_soft"]["passive_host_checks_are_soft"] != 2 ?
        $rq .= "'".$ret["passive_host_checks_are_soft"]["passive_host_checks_are_soft"]."', " : $rq .= "'2', ";
    isset($ret["check_for_orphaned_hosts"]["check_for_orphaned_hosts"])
        && $ret["check_for_orphaned_hosts"]["check_for_orphaned_hosts"] != 2 ?
        $rq .= "'".$ret["check_for_orphaned_hosts"]["check_for_orphaned_hosts"]."', " : $rq .= "'2', ";
    isset($ret["external_command_buffer_slots"]["external_command_buffer_slots"])
        && $ret["external_command_buffer_slots"]["external_command_buffer_slots"] != 2 ?
        $rq .= "'".$ret["external_command_buffer_slots"]["external_command_buffer_slots"]."', " : $rq .= "'2', ";
    isset($ret["log_pid"]["log_pid"]) && $ret["log_pid"]["log_pid"] ? $rq .= "'1', " : $rq .= "'0', ";
    if (isset($ret["nagios_activate"]["nagios_activate"]) && $ret["nagios_activate"]["nagios_activate"]) {
    isset($ret["enable_notifications"]["enable_notifications"])
        && $ret["enable_notifications"]["enable_notifications"] != 2 ?
        . $ret["enable_notifications"]["enable_notifications"]."',  "
    isset($ret["execute_service_checks"]["execute_service_checks"])
        && $ret["execute_service_checks"]["execute_service_checks"] != 2 ?
        . $ret["execute_service_checks"]["execute_service_checks"]."',  "
    isset($ret["accept_passive_service_checks"]["accept_passive_service_checks"])
        && $ret["accept_passive_service_checks"]["accept_passive_service_checks"] != 2 ?
        . $ret["accept_passive_service_checks"]["accept_passive_service_checks"]."',  "
    isset($ret["execute_host_checks"]["execute_host_checks"])
        && $ret["execute_host_checks"]["execute_host_checks"] != 2 ?
        $rq .= "execute_host_checks = '".$ret["execute_host_checks"]["execute_host_checks"]."',  "
    isset($ret["accept_passive_host_checks"]["accept_passive_host_checks"])
        && $ret["accept_passive_host_checks"]["accept_passive_host_checks"] != 2 ?
        . $ret["accept_passive_host_checks"]["accept_passive_host_checks"]."',  "
    isset($ret["enable_event_handlers"]["enable_event_handlers"])
        && $ret["enable_event_handlers"]["enable_event_handlers"] != 2 ?
        . $ret["enable_event_handlers"]["enable_event_handlers"]."',  "
    isset($ret["log_rotation_method"]["log_rotation_method"])
        && $ret["log_rotation_method"]["log_rotation_method"] != 2 ?
        . $ret["log_rotation_method"]["log_rotation_method"]."',  "
    isset($ret["check_external_commands"]["check_external_commands"])
        && $ret["check_external_commands"]["check_external_commands"] != 2 ?
        . $ret["check_external_commands"]["check_external_commands"]."',  "
    isset($ret["retain_state_information"]["retain_state_information"])
        && $ret["retain_state_information"]["retain_state_information"] != 2 ?
        . $ret["retain_state_information"]["retain_state_information"]."',  "
    isset($ret["use_retained_program_state"]["use_retained_program_state"])
        && $ret["use_retained_program_state"]["use_retained_program_state"] != 2 ?
        . $ret["use_retained_program_state"]["use_retained_program_state"]."',  "
    isset($ret["use_retained_scheduling_info"]["use_retained_scheduling_info"])
        && $ret["use_retained_scheduling_info"]["use_retained_scheduling_info"] != 2 ?
        . $ret["use_retained_scheduling_info"]["use_retained_scheduling_info"]."',  "
    isset($ret["use_syslog"]["use_syslog"]) && $ret["use_syslog"]["use_syslog"] != 2 ?
        $rq .= "use_syslog = '".$ret["use_syslog"]["use_syslog"]."',  "
    isset($ret["log_notifications"]["log_notifications"])
        && $ret["log_notifications"]["log_notifications"] != 2 ?
        $rq .= "log_notifications = '".$ret["log_notifications"]["log_notifications"]."',  "
    isset($ret["log_service_retries"]["log_service_retries"])
        && $ret["log_service_retries"]["log_service_retries"] != 2 ?
        . $ret["log_service_retries"]["log_service_retries"]."',  "
    isset($ret["log_host_retries"]["log_host_retries"])
        && $ret["log_host_retries"]["log_host_retries"] != 2 ?
        $rq .= "log_host_retries = '".$ret["log_host_retries"]["log_host_retries"]."',  "
    isset($ret["log_event_handlers"]["log_event_handlers"])
        && $ret["log_event_handlers"]["log_event_handlers"] != 2 ?
        $rq .= "log_event_handlers = '".$ret["log_event_handlers"]["log_event_handlers"]."',  "
    isset($ret["log_external_commands"]["log_external_commands"])
        && $ret["log_external_commands"]["log_external_commands"] != 2 ?
        $rq .= "log_external_commands = '".$ret["log_external_commands"]["log_external_commands"]."',  "
    isset($ret["log_passive_checks"]["log_passive_checks"])
        && $ret["log_passive_checks"]["log_passive_checks"] != 2 ?
        $rq .= "log_passive_checks = '".$ret["log_passive_checks"]["log_passive_checks"]."',  "
    isset($ret["auto_reschedule_checks"]["auto_reschedule_checks"])
        && $ret["auto_reschedule_checks"]["auto_reschedule_checks"] != 2 ?
        $rq .= "auto_reschedule_checks = '".$ret["auto_reschedule_checks"]["auto_reschedule_checks"]."', "
    isset($ret["use_aggressive_host_checking"]["use_aggressive_host_checking"])
        && $ret["use_aggressive_host_checking"]["use_aggressive_host_checking"] != 2 ?
        . $ret["use_aggressive_host_checking"]["use_aggressive_host_checking"]."',  "
    isset($ret["enable_predictive_host_dependency_checks"]["enable_predictive_host_dependency_checks"])
        && $ret["enable_predictive_host_dependency_checks"]["enable_predictive_host_dependency_checks"] != 2 ?
        . $ret["enable_predictive_host_dependency_checks"]["enable_predictive_host_dependency_checks"]."',  "
    isset($ret["enable_flap_detection"]["enable_flap_detection"])
        && $ret["enable_flap_detection"]["enable_flap_detection"] != 2 ?
        $rq .= "enable_flap_detection = '".$ret["enable_flap_detection"]["enable_flap_detection"]."',  "
    isset($ret["soft_state_dependencies"]["soft_state_dependencies"])
        && $ret["soft_state_dependencies"]["soft_state_dependencies"] != 2 ?
        $rq .= "soft_state_dependencies   = '".$ret["soft_state_dependencies"]["soft_state_dependencies"]."',  "
    isset($ret["enable_predictive_service_dependency_checks"]["enable_predictive_service_dependency_checks"])
        && $ret["enable_predictive_service_dependency_checks"]["enable_predictive_service_dependency_checks"] != 2 ?
        . $ret["enable_predictive_service_dependency_checks"]["enable_predictive_service_dependency_checks"]."',  "
    isset($ret["obsess_over_services"]["obsess_over_services"])
        && $ret["obsess_over_services"]["obsess_over_services"] != 2 ?
        . $ret["obsess_over_services"]["obsess_over_services"]."',  "
    isset($ret["obsess_over_hosts"]["obsess_over_hosts"])
        && $ret["obsess_over_hosts"]["obsess_over_hosts"] != 2 ?
        $rq .= "obsess_over_hosts = '".$ret["obsess_over_hosts"]["obsess_over_hosts"]."',  "
    isset($ret["process_performance_data"]["process_performance_data"])
        && $ret["process_performance_data"]["process_performance_data"] != 2 ?
        . $ret["process_performance_data"]["process_performance_data"]."',  "
    isset($ret["host_perfdata_file_mode"]["host_perfdata_file_mode"])
        && $ret["host_perfdata_file_mode"]["host_perfdata_file_mode"] != null ?
        . $ret["host_perfdata_file_mode"]["host_perfdata_file_mode"]."',  "
    isset($ret["service_perfdata_file_mode"]["service_perfdata_file_mode"])
        && $ret["service_perfdata_file_mode"]["service_perfdata_file_mode"] != null ?
        . $ret["service_perfdata_file_mode"]["service_perfdata_file_mode"]."',  "
    isset($ret["check_for_orphaned_services"]["check_for_orphaned_services"])
        && $ret["check_for_orphaned_services"]["check_for_orphaned_services"] != 2 ?
        . $ret["check_for_orphaned_services"]["check_for_orphaned_services"]."',  "
    isset($ret["check_service_freshness"]["check_service_freshness"])
        && $ret["check_service_freshness"]["check_service_freshness"] != 2 ?
        . $ret["check_service_freshness"]["check_service_freshness"]."',  "
    isset($ret["check_host_freshness"]["check_host_freshness"])
        && $ret["check_host_freshness"]["check_host_freshness"] != 2 ?
        . $ret["check_host_freshness"]["check_host_freshness"]."',  "
    isset($ret["use_large_installation_tweaks"]["use_large_installation_tweaks"])
        && $ret["use_large_installation_tweaks"]["use_large_installation_tweaks"] != 2 ?
        . $ret["use_large_installation_tweaks"]["use_large_installation_tweaks"]."',  "
    isset($ret["enable_environment_macros"]["enable_environment_macros"])
        && $ret["enable_environment_macros"]["enable_environment_macros"] != 2 ?
        . $ret["enable_environment_macros"]["enable_environment_macros"]."',  "
    isset($ret["use_setpgid"]["use_setpgid"]) && $ret["use_setpgid"]["use_setpgid"] != 2 ?
        $rq .= "use_setpgid = '".$ret["use_setpgid"]["use_setpgid"]."',  "
    isset($ret["use_regexp_matching"]["use_regexp_matching"])
        && $ret["use_regexp_matching"]["use_regexp_matching"] != 2 ?
        $rq .= "use_regexp_matching = '".$ret["use_regexp_matching"]["use_regexp_matching"]."',  "
    isset($ret["use_true_regexp_matching"]["use_true_regexp_matching"])
        && $ret["use_true_regexp_matching"]["use_true_regexp_matching"] != 2 ?
        . $ret["use_true_regexp_matching"]["use_true_regexp_matching"]."',  "
    isset($ret["debug_verbosity"]["debug_verbosity"])
        && $ret["debug_verbosity"]["debug_verbosity"] != 2 ?
        $rq .= "debug_verbosity   = '".$ret["debug_verbosity"]["debug_verbosity"]."',  "
    isset($ret["daemon_dumps_core"]["daemon_dumps_core"])
        && $ret["daemon_dumps_core"]["daemon_dumps_core"] ?
    isset($ret["translate_passive_host_checks"]["translate_passive_host_checks"])
        && $ret["translate_passive_host_checks"]["translate_passive_host_checks"] != null ?
        . htmlentities($ret["translate_passive_host_checks"]["translate_passive_host_checks"], ENT_QUOTES, "UTF-8")
    isset($ret["passive_host_checks_are_soft"]["passive_host_checks_are_soft"])
        && $ret["passive_host_checks_are_soft"]["passive_host_checks_are_soft"] != null ?
        . htmlentities($ret["passive_host_checks_are_soft"]["passive_host_checks_are_soft"], ENT_QUOTES, "UTF-8")
    isset($ret["check_for_orphaned_hosts"]["check_for_orphaned_hosts"])
        && $ret["check_for_orphaned_hosts"]["check_for_orphaned_hosts"] != null ?
        . htmlentities($ret["check_for_orphaned_hosts"]["check_for_orphaned_hosts"], ENT_QUOTES, "UTF-8")
    isset($ret["log_pid"]["log_pid"]) && $ret["log_pid"]["log_pid"] ?
    $rq .= "nagios_activate = '".$ret["nagios_activate"]["nagios_activate"]."' ";
    if ($ret["nagios_activate"]["nagios_activate"]) {

```

**Comment by lpinsivy** [comment]
Corrected by #7591
